### ISSUE TITLE
Convert `/pathogens` to App Router

### ISSUE THREAD

**Issue by genehack** [nextstrain/nextstrain.org#1066]
### Context
This is part of the [Convert static-site from Pages Router to App Router](https://github.com/nextstrain/nextstrain.org/issues/1052) work. See there for additional context.

The `/pathogens` page uses only the `GenericPage` and `ListResources` components.

The `GenericPage` component is a simple bit of HTML and will be trivial to convert.

`ListResources` is a fairly massive component that uses some client-side state. On the plus side, it depends on very few things outside itself, and it has already been rewritten in Typescript (albeit, still making extensive use of Styled Components). Converting this component over is expected to be the bulk of the work.

**Comment by victorlin** [comment]
I have some ListResources TypeScript work that I plan to make a PR for soon. Hopefully the app router work can wait for that.

**Comment by genehack** [comment]
> I have some ListResources TypeScript work that I plan to make a PR for soon. Hopefully the app router work can wait for that.

yeah, i'm gonna jump back into the blog PR and deal with the redirect issue there before i pick this back up — yesterday i got right to the point where i was gonna touch ListResources and ran out of day, so once that other PR is merged, i can rebase my WIP on top of it.

**Comment by victorlin** [comment]
PR for the TypeScript work: https://github.com/nextstrain/nextstrain.org/pull/1073
